td-shim-interface/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "td-shim-interface"
+version = "0.1.0"
+license = "BSD-2-Clause-Patent"
+description = "TD-shim metadata data structures and related functions. UEFI Platform Initializaiton data structures and accessors"
+
+edition = "2018"
+homepage = "https://github.com/confidential-containers/td-shim"
+repository = "https://github.com/confidential-containers/td-shim"
+readme = "README.md"
+keywords = ["td-shim", "TDX", "intel"]
+
+[build-dependencies]
+anyhow = "1.0.55"
+cc = "1.0"
+which = "5.0.0"
+
+[dependencies]
+lazy_static = { version = "1.4.0", features = ["spin_no_std"] }
+r-efi = "3.2.0"
+scroll = { version = "0.10", default-features = false, features = ["derive"] }
+zerocopy = { version = "0.7.31", features = ["derive"] }
+
+x86 = { version ="0.47.0", optional = true }
```

Comment:
Are these crates required by the new crate? I think we can find out and remove the unused dependencies

Reply:
Already clean the unused dependencies. Thanks!